File: Logic-1/love6/Love6.java
package love6;

public class Love6 {
    //Retorne true se "a" ou "b" forem 6, ou se a soma resulta no mesmo, ou até mesmo se a sua diferença também o for.

    public boolean love6(int a, int b) {
        if (a == 6 || b == 6) return true;
        else return  (a+b == 6 || Math.abs(a-b) == 6);
    }
}

File: Python/Logic-1/love6.py
"""
Dado 2 números inteiros ("a" e "b"), retorne true caso eles sejam "6" ou se sua
soma ou diferença absoluta resulte em 6.
"""


def love6(a, b):
    return a == 6 or b == 6 or a+b == 6 or abs(a-b) == 6


print(love6(4, 2))
